src/Microsoft.AspNet.Mvc.ModelBinding/ModelMetadata.cs

```diff
@@ -167,7 +167,17 @@ public string PropertyName
         public abstract bool HideSurroundingHtml { get; }
 
         /// <summary>
-        /// Gets a value indicating whether or not the model value is required by model binding. This is only
+        /// Gets a value indicating whether or not the model value can be bound by model-binding. This is only
+        /// applicable when the current instance represents a property.
+        /// </summary>
+        /// <remarks>
+        /// If <c>true</c> then the model value is considered supported by model-binding and can be set
+        /// based on provided input in the request.
+        /// </remarks>
+        public abstract bool IsBindingAllowed { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether or not the model value is required by model-binding. This is only
```

Comment:
sure. I think I felt like using *model-binding* one day and started doing it :+1: I'll clean it up.